src/libraries/iKin/src/iKinFwd.cpp

```diff
@@ -2711,8 +2711,8 @@ void iCubInertialSensor::allocate(const string &_type)
         setHN(getHN()*HN);
     }
 
-    // further displacement for v2.6
-    if (getType()=="v2.6")
+    // further displacement for > 2.6
+    if (version >= 2.6)
```

Comment:
```suggestion
    if (version>=2.6)
```

just for consistency in formatting